src/components/PhotosGallery/PhotosGallery.jsx

```diff
@@ -0,0 +1,124 @@
+import Image from "next/image"
+import React, { useEffect, useState } from "react"
+
+function PhotosGallery(props) {
+  const [file, setFile] = useState(null)
+
+  const onChangeImageHandler = (e) => {
+    if (e.target.files[0]) {
+      const reader = new FileReader()
+      reader.onload = () => {
+        setFile(reader.result)
+      }
+      reader.readAsDataURL(e.target.files[0])
+      e.target.value = ""
+    }
+  }
+
+  useEffect(() => {}, [file])
+
+  function dropLargeImage() {
+    return (
+      <div className='w-full'>
+        <div className='border border-solid bg-gray-300 border-black rounded relative '>
+          <input
+            type='file'
+            className='cursor-pointer relative opacity-0 w-full p-14 lg:p-24 md:p-24'
+            onChange={onChangeImageHandler}
+          ></input>
+          <div className='w-12 absolute top-10 md:top-20 lg:top-20 right-0 left-0 m-auto'>
+            <Image
+              src='gallery.svg'
```

Comment:
Instead of hard-coding the string "gallery.svg", you can store it in a constant like const GALLERY_SVG = 'gallery.svg' and use it in both "dropLargeImage" and "dropImage" by replacing the hard-coded string with {galleryImg}.